WindowsLib/RichEditControl.cpp

```diff
@@ -48,7 +48,7 @@ namespace WinLib {
         if ( scrollable )
             dwStyle |= ES_AUTOVSCROLL|ES_AUTOHSCROLL|WS_VSCROLL|WS_HSCROLL;
 
-        return WindowControl::CreateControl( WS_EX_CLIENTEDGE, MSFTEDIT_CLASS, "", dwStyle,
+        return WindowControl::CreateControl( WS_EX_CLIENTEDGE, RICHEDIT_CLASS, "", dwStyle,
```

Comment:
MSFTEDIT_CLASS is not a typo, it's the newest version of rich edit controls [Versions of Rich Edit](https://learn.microsoft.com/en-us/windows/win32/controls/about-rich-edit-controls#versions-of-rich-edit), was this not working for you? if so can I get a little context as to what you're compiling on/compiling for?